operations/image_management/Upload_and_Register_an_Image_Recipe.md

```diff
@@ -131,3 +131,145 @@ Download and expand recipe archives from S3 and IMS. Modify and upload a recipe
     type = "s3"
     ```
 
+## Templating IMS recipes when building an IMS image
+
+IMS can optionally template the contents of an IMS recipe before building the recipe with kiwi-ng during an IMS create job. This 
+enables variables stored in the recipe to be dynamically replaced with values registered with the IMS recipe record after the 
+recipe is downloaded from S3 but before it is built by Kiwi-ng. This functionality is used by various Cray products to ensure 
+that the resulting image is built from the most correct release versions of Nexus hosted RPM repositories. 
+
+Follow the steps below to enable IMS templating within a given recipe. Note that for IMS to properly template a recipe,
+the following procedures must all be completed.
+
+### Enable templating in IMS recipe archive tgz file
```

Comment:
Change to:

Enable Templating in IMS Recipe Archive tgz File

Reply:
Done